app.js

```diff
@@ -32,14 +32,14 @@ app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'html');
 
 app.use(session({
-	secret: auth.key, 
-	saveUninitialized: true,
-	resave: true,
-	store: new MongoStore({
-		url: auth.url,
-		ttl: 24 * 60 * 60,
-		touchAfter: 5 * 60
-	})
```

Comment:
Why are you changing the indentation everywhere? The entire project is in tabs...